sql/dql/intro-joins/primary-foreign-keys.md

```diff
@@ -0,0 +1,93 @@
+---
+author: meridian
+
+type: normal
+
+category: must-know
+
+links: 
+  - >-
+   [Difference between Primary and Foreign 
+   Keys]
+   (https://www.dotnettricks.com/learn/sqlserver/difference-between-primary-key-and-foreign-key){article}
+
+tags:
+  - introduction
+  - joins
+
+practiceQuestion:
+  formats:
+    - fill-in-the-gap
+    - type-in-the-gap
```

Comment:
in general, we shouldn't mark text-only questions as type-in-the-gap since there are multiple words that could be used to answer but we are only able to check for a specific one
```suggestion
```